### DISCUSSION TITLE
Disabling Dialog display for storage type

### DISCUSSION THREAD

**Discussion by lcetinsoy** [jgraph/drawio#3696] [Q&A]
Hello ! 

I have developed a plugin for diagrams which loads a xml graph from the parent iframe via the postMessage api. 
I listen to the init event in order to load the graph only when diagrams is ready.
My problem is that the storage Dialog is triggered after the init event and override the graph that has been loaded. 
What would you suggest to circumvent the problem ? 

I tried to disable the Dialog display (taking inspiration from trello.js plugin) without much success so far by adding the following line to my plugin : 

```js
jseditorUi.showDialog = function() {};
```

I also tried to hardcode somewhere a default storage mode without success (in my fork of the code)  ^^

**Comment by davidjgraph** [comment]
If you're using in embed mode, the storage dialog shouldn't be displayed. If you're not using embed=1, maybe try that?

**Comment by lcetinsoy** [comment]
Thank you for your feedback ! I managed to make it work as I wanted using  the editorUi.showDialog = function() {} (without embed=1)
next time I will try embed=1
Best !
